Content.Server/Shuttles/Systems/ThrusterSystem.cs

```diff
@@ -518,7 +504,7 @@ private void OnUpgradeExamine(EntityUid uid, ThrusterComponent component, Upgrad
 
         #endregion
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
```

Comment:
is it necessary to use the whole namespace like this?

Reply:
`System.Runtime.CompilerServices` also has `DependencyAttribute`, so that conflicts with the EntitySystem `Dependency`s at the top of the file. I swapped it because I preferred to use the full namespace here once, over six times at the top.